extensions/amp-analytics/0.1/vendors.js

```diff
@@ -297,5 +297,29 @@ export const ANALYTICS_CONFIG = {
       'pageview': 'https://${host}${requestPath}?${basePrefix}',
       'click': 'https://${host}${requestPath}?${basePrefix}&pe=lnk_${linkType}&pev1=${linkUrl}&pev2=${linkName}',
     }
+  },
+
+  'infonline': {
+    'vars': {
+      "sv": "ke",
+      "ap": "1",
+      "url": "https://3p.ampproject.net/custom/amp-analytics-infonline.html"
+    },
+    'transport': {'beacon': false, 'xhrpost': false, 'image': true},
+    'requests': {
+      'pageview': '${url}?st=${st}' +
+        '&sv=${sv}' +
+        '&ap=${ap}' +
+        '&co=${co}' +
+        '&cp=${cp}' 
+    },
+    'triggers': {
+      'pageview': {
+        'on': 'visible',
+        'request': 'pageview',
+        'iframe' + /* TEMPORARY EXCEPTION */ + 'Ping'
```

Comment:
Delete this line and add this to the end of the file:

`ANALYTICS_CONFIG['infonline']['triggers']['pageview'][
    'iframe' + /* TEMPORARY EXCEPTION */ + 'Ping'] = true;`